tools/dev/run_dev_server.sh

```diff
@@ -42,7 +42,7 @@ case $CONFIG in
       AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED=true \
       DEFAULT_VECTORIZER_MODULE=text2vec-contextionary \
       BACKUP_FILESYSTEM_PATH="${PWD}/backups" \
-      ENABLE_MODULES="text2vec-contextionary,backup-filesystem" \
+      ENABLE_MODULES="text2vec-contextionary,cross-ranker-transformers" \
```

Comment:
@CShorten this should be replaced with a dedicated option similar to how `local-centroid` is done